java/src/main/java/be/tothepoint/model/projections/NrOfPlayersProjection.java

```diff
@@ -0,0 +1,22 @@
+package be.tothepoint.model.projections;
+
+import java.util.List;
+
+import be.tothepoint.model.Event;
+
+public class NrOfPlayersProjection implements Projection<Long> {
+
+  @Override
+  public Long project(List<Event> events) {
```

Comment:
I'd like to only have one implementation on all technology stacks: count the events. The implementation should be as simple as possible. 

So for imperative OO languages with a loop or a simple .count() on the list. Like in [the csharp example](https://github.com/michelgrootjans/playing_with_projections/blob/master/csharp/PlayingWithProjections/Program.cs#L39): `events.Count();`

For functional languages, a functional implementation is obviously the choice.